website/client/store/actions/auth.js

```diff
@@ -68,5 +68,5 @@ export async function socialAuth (store, params) {
 export function logout () {
   localStorage.removeItem(LOCALSTORAGE_AUTH_KEY);
   localStorage.removeItem(LOCALSTORAGE_SOCIAL_AUTH_KEY);
-  window.location.href = '/logout';
+  window.location.href = '/logoutServer';
```

Comment:
Can you turn this in an axios request? We don't want to redirect the users to the server page, right @paglias?

Reply:
Ok. This shouldn't be too hard.

Reply:
actually it should be a redirect , the server will redirect to the home page